.github/workflows/main.yml

```diff
@@ -1,25 +1,59 @@
-name: Dokku CD
+name: main
 on:
   push:
     branches: [main]
 env:
-  DOKKU_APP_NAME: "outbreak"
-  DOKKU_HOST: "dokku-1.hurumap.org"
   DOKKU_REMOTE_BRANCH: "master"
+  DOKKU_REMOTE_URL: "ssh://dokku@dokku-1.hurumap.org/outbreak"
```

Comment:
I think we can make this configurable instead of hardcoding the URL.

Reply:
@kilemensi  I think using `${{ env.DOKKU_REMOTE_URL }}` as used below should also work here.

Reply:
Not sure I get you @esirK, I'm _declaring_ the env here. Or do you mean we should use GitHub secret for this?